src/Helper/Quote.php

```diff
@@ -152,17 +152,28 @@ protected function setItemQty()
     }
 
     /**
-     * Assigns shipping method to the quote
+     * If it's an online method, the default Shopgate payment method is used as it will not trigger capture when the
+     * quote turns into an order. We manipulate the online payment method after the order is created.
      *
      * @throws LocalizedException
+     * @throws \ShopgateLibraryException
```

Comment:
I'd figure it would not have the backwards slash if it is imported above?
```suggestion
     * @throws ShopgateLibraryException
```